presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/api/ControllerConfigAPI.java

```diff
@@ -37,8 +42,10 @@
 import static org.eclipse.jetty.http.HttpMethod.DELETE;
 import static org.eclipse.jetty.http.HttpMethod.GET;
 import static org.eclipse.jetty.http.HttpMethod.POST;
+import static org.eclipse.jetty.http.HttpMethod.PUT;
 
 @Path("/config")
+@Api(description = "the controller config API")
```

Comment:
This isn't a useful description. Maybe something like this:

"API to modify Presto Configuration throughout the cluster"